boards/BMS/Src/App/states/App_ChargeState.c

```diff
@@ -44,23 +50,32 @@ static void ChargeStateRunOnTick100Hz(struct StateMachine *const state_machine)
             has_charger_faulted = App_Charger_HasFaulted(charger);
         }
 
-        uint8_t    segment = 0U;
-        uint8_t    cell    = 0U;
-        const bool has_reached_max_v =
-            App_Accumulator_GetMaxVoltage(accumulator, &segment, &cell) > MAX_CELL_VOLTAGE_THRESHOLD;
+        uint8_t    segment           = 0U;
+        uint8_t    cell              = 0U;
+        const bool has_reached_max_v = App_Accumulator_GetMaxVoltage(accumulator, &segment, &cell) > MAX_CELL_V_TARGET;
+        App_CanTx_SetPeriodicSignal_IS_CHARGING_COMPLETE(can_tx, has_reached_max_v);
```

Comment:
maybe should move this to when the pack is balanced and we have reached max voltage